http/src/main/resources/reference.conf

```diff
@@ -39,6 +39,7 @@ dataproc {
 }
 
 gce {
+  region = "us-central1"
```

Comment:
Should we rename the [Dataproc counterpart](https://github.com/DataBiosphere/leonardo/pull/1319/files#diff-1f2b53dcb909be51bfea49ec1b078e61R14) `region` as well so they are consistent? Also, we only support one region so `default` in the name might be misleading.

Reply:
That makes sense.. I'm going to have to rebase https://github.com/DataBiosphere/leonardo/pull/1309 so I can do it there

Reply:
👍